pkg/controller/storageprofile-controller.go

```diff
@@ -35,6 +37,7 @@ var (
 			Name: monitoring.MetricOptsList[monitoring.IncompleteProfile].Name,
 			Help: monitoring.MetricOptsList[monitoring.IncompleteProfile].Help,
 		})
+	scProvisionerMap = make(map[string]string)
```

Comment:
this map will reconstruct properly if the controller dies right? just making sure since storing these things in pods is usually not what we do

Reply:
It is pretty unusual, I am doing it to filter the PV changes that trigger the reconcile, and I am under the impression it should run very fast.
I am not 100% sure what happens when the controller dies, but as long as it runs once for the correct storage class (and it probably does) it'll build the correct storage capabilities. I don't mind the missed PV updates.

Reply:
Done